fusion_docs/guide.mdx

```diff
@@ -6,14 +6,15 @@ title: User guide
 
 Fusion is a virtual, lightweight, distributed file system designed to optimise the data access of Nextflow data pipelines.
 
-Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler
-pipeline logic and faster, more efficient pipeline execution.
+Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler pipeline logic and faster, more efficient pipeline execution.
+
+:::note
+Fusion is free for trial purposes. To use Fusion in a production environment, a license is required. Fusion is licensed by Seqera for use in Seqera Platform or directly with Nextflow.
```

Comment:
Same note as before. Fusion is free to evaluate, but what constitutes production use is nebulous, and largely irrelevant. If we can state the exact terms of the evaluation period, we can simplify this.

Reply:
"Fusion is free to evaluate for X (time/pipeline runs/GB data/etc.). Contact us for a license to use Fusion in Seqera Platform or Nextflow."

Reply:
Very much agree with Llewellyn. I don't want people to think "production use" means running production pipelines - if they are using Fusion to iteratively develop pipelines, that is still non-trial / use of Fusion. 

The limits of the free usage have now been defined (though not technically implemented) - should we note them here now or is it too premature? Going to talk to drew about this in ~30

Reply:
OR! something like this 

Fusion requires a license for use within Seqera Platform or with Nextflow outside of Seqera Platform. Try Fusion for free by following the steps below:

Reply:
I'd like to avoid the specific numbers until they're implemented, but this slight adjustment on what you proposed works for me – "Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. Contact Seqera for more details."